pages.es/osx/rargs.md

```diff
@@ -0,0 +1,25 @@
+# rargs
+
+> Ejecuta un comando por cada línea de entrada estándar.
+> Como `xargs`, pero con soporte de coincidencia de patrones.
+> Más información: <https://github.com/lotabout/rargs>.
+
+- Ejecuta un comando por cada línea de entrada, como `xargs` (`{0}` indica dónde sustituir en el texto):
+
+`{{command}} | rargs {{command}} {0}`
+
+- Realiza una ejecución en seco, lo cual imprime los comandos que se ejecutarían en lugar de ejecutarlos:
```

Comment:
This suggestion may be skipped because I am not sure if `ejecución en seco` is a valid term. Although, I did find a [website](https://estudyando.com/pruebas-de-ejecucion-en-seco-en-el-desarrollo-de-software) using the term.

```suggestion
- Imprime los comandos que se ejecutarían en vez de ejecutarlos:
```